docs/api/go.md

```diff
@@ -27,6 +27,40 @@ import (
 )
 ```
 
+## Appender
```

Comment:
Excellent - we should use the `README` contents of this PR: https://github.com/marcboeker/go-duckdb/pull/154 once it is merged, as the current `README` will be outdated.